airflow/models/mappedoperator.py

```diff
@@ -386,6 +386,10 @@ def leaves(self) -> Sequence[AbstractOperator]:
         """Implementing DAGNode."""
         return [self]
 
+    @property
+    def task_display_name(self) -> str:
+        return self.partial_kwargs.get("task_display_name", None) or self.task_id
```

Comment:
``
```suggestion
        return self.partial_kwargs.get("task_display_name") or self.task_id
```

Reply:
Mhm I explicitly left the fallback to `None`in because I was not sure if we can assume the task_display_name is in always. Did not want to risk a KeyError.
Trying locally if removal of `None` as default has a harm in pytest and make a small regression before pushing this away

Reply:
Seems safe.